File: src/main/webapp/js/service/questionService.js
app.service('questionService',function($http){
	this.submitQuestion=function(question){
		return $http.post("./question/add",question);
	}
	this.getQuestionList=function(pageNum,pageSize){
		return $http.get("./question/findList?pageNum="+pageNum+"&pageSize="+pageSize);
	}
	this.getDetail=function(id,pageNum,pageSize){
		return $http.get("./question/findOne?id="+id+"&pageNum="+pageNum+"&pageSize="+pageSize)
	}
	this.reply=function(comment){
		return $http.post('./comment/add',comment);
	}
})

File: src/main/java/com/github/qia/service/QuestionService.java
package com.github.qia.service;

import com.github.qia.pojo.Question;
import com.github.qia.pojogroup.FullQuestion;

import entity.PageResult;

public interface QuestionService {

	void add(Question question);

	PageResult findList(int pageNum, int pageSize);
	
	FullQuestion findOne(Long id,int pageNum,int pageSize);

	PageResult findAll();

}
